pkg/server/server.go

```diff
@@ -33,7 +33,7 @@ type NilServerListError struct {
 }
 
 func (e *NilServerListError) Error() string {
-	return fmt.Sprintf("%v: server error", "Server list are nil")
+	return fmt.Sprintf("%v: server error", "Server list are nil and maps in golang are useless when they are nil. Reference : https://blog.golang.org/go-maps-in-action")
```

Comment:
Plusieurs server list --> are
Une seule --> is

golang doit débuter avec une majuscule? (nom propre)